_schedule/tutorials/2019-09-22-13-30-t1-django-bootstrapping-done-right.md

```diff
@@ -0,0 +1,69 @@
+---
+abstract: A step by step guide to setup a Django project with all the required tools
+  you will need. We will cover all aspects a project must have, from the development
+  environment with Docker to automated deployment with Pipelines.
+accepted: true
+category: tutorials
+date: 2019-09-22 13:30:00 -0500
+difficulty: Beginner
+end_date: 2019-09-22 17:00:00 -0500
+layout: session-details
+permalink: /tutorials/django-bootstrapping-done-right/
+presenters:
+- pablo-garcia
+published: true
+room: Sierra 5
+sitemap: true
+slides_url: ''
+summary: ''
+tags:
+- django
+- docker
+- ci
+- tools
+title: Django bootstrapping done right
+track: t1
+video_url: ''
+---
+
+# Django bootstrapping done right
```

Comment:
We can probably remove this line as we already display the title above
<img width="1142" alt="Screen Shot 2019-06-29 at 12 16 49 PM" src="https://user-images.githubusercontent.com/3345131/60386791-d35e6a80-9a67-11e9-9664-5f9becf9046e.png">